README.md

```diff
@@ -18,12 +18,37 @@ We welcome contributions and feedback. Please follow our [contributing guideline
 
 ---
 
-## To run locally
+## one-time setup
+- clone this repo
+- run **npm install**
 
-- Clone this repository
+## To run locally
 - Ensure that you are at the solution folder
```

Comment:
To run locally in your SharePoint workbench

Reply:
I changed it to say "To run locally in your SharePoint Workbench, or to support local builds"

Reply:
maybe that should be separate?
"Support local builds"
- this would be on-time setup like the gulp cert
"Run locally in SharePoint Workbench"
- npm install
- npm serve
-